### PR TITLE
Homework1: Sergey Streltsov

### PR DESCRIPTION
**Pull request by Mantikor**
i have moved all examples to one file, maybe that's wrong decision

### CODE REVIEW

File: src/homework01.py [hunk 1]
```diff
@@ -0,0 +1,327 @@
...
+friends = ['john', 'pat', 'gary', 'michael']
+for i, name in enumerate(friends):
+    print('iteration {iteration} is {name}'.format(iteration=i, name=name))
+
+# 4 lines: Fibonacci, tuple assignment
+parents, babies = (1, 1)
+while babies < 100:
+    print('This generation has {0} babies'.format(babies))
+    parents, babies = (babies, parents + babies)
+
```
Comment (belovmd) [inline src/homework01.py:43]: why previous tasks were separated with one empty line and this one with two?

File: src/homework01.py [hunk 2]
```diff
@@ -0,0 +1,327 @@
...
+
+# 5 lines: Functions
+def greet(name):
+    print('Hello', name)
+
+
```
Comment (belovmd) [inline src/homework01.py:49]: why this function's calls are separated from this function with 2 lines and next task with one from calls?

File: src/homework01.py [hunk 3]
```diff
@@ -0,0 +1,327 @@
...
+%d bottles of beer on the wall,
+%d bottles of beer,
+take one down, pass it around,
+%d bottles of beer on the wall!
+'''
+bottles_of_beer = 9
+while bottles_of_beer > 1:
+    print(REFRAIN % (bottles_of_beer, bottles_of_beer, bottles_of_beer - 1))
+    bottles_of_beer -= 1
+
```
Comment (belovmd) [inline src/homework01.py:120]: why here we have 2 lines between tasks and in previous one?

File: src/homework01.py [hunk 4]
```diff
@@ -0,0 +1,327 @@
...
+import random
+import re
+import sys
+from time import localtime
+import unittest
+import xml.etree.ElementTree as etree
+
+# 1 line: Output
+print('Hello, world!')
+
```
Comment (belovmd) [inline src/homework01.py:25]: it looks as a good idea to separate tasks from each other with 2 empty lines everywhere

File: src/homework01.py [hunk 5]
```diff
@@ -0,0 +1,327 @@
...
+This is the second.
+'''.splitlines()
+# Use itertools.groupby and bool to return groups of
+# consecutive lines that either have content or don't.
+for has_chars, frags in groupby(lines, bool):
+    if has_chars:
+        print(' '.join(frags))
+
+
+# PRINTS:
```
Comment (belovmd) [inline src/homework01.py]: do we need this comments here?

**Review by belovmd — CHANGES_REQUESTED** [review]
This code is OK itself but looks pretty weird. Please address issues below
and please fix issues like that everywhere

File: src/homework01.py (hunk 2 above)
Comment (Mantikor) [reply src/homework01.py:49]: this recommendation comes from PyCharm, PEP8: expected 2 blank lines, found 1, i think this problem is due i join all tasks in one file

**Review by belovmd — COMMENTED** [review]
8:
1.  unnecessary files (from 2nd homework)
2. used master branch (in guidelines were suggested to use feature branch)

**Pull request closed without merging**